#! /bin/sh /usr/share/dpatch/dpatch-run
## postgresql_conf_fix.dpatch by Sebastian Harl <sh@tokkee.org>
##
## DP: postgresql_default.conf: Don't use the deprecated {Min,Max}PgVersion
## DP: options.

@DPATCH@

diff a/src/postgresql_default.conf b/src/postgresql_default.conf
--- a/src/postgresql_default.conf
+++ b/src/postgresql_default.conf
@@ -54,7 +54,7 @@
 		ValuesFrom "del"
 	</Result>
 
-	MaxPGVersion 80299
+	MaxVersion 80299
 </Query>
 
 <Query queries>
@@ -85,7 +85,7 @@
 		ValuesFrom "hot_upd"
 	</Result>
 
-	MinPGVersion 80300
+	MinVersion 80300
 </Query>
 
 <Query query_plans>
@@ -132,7 +132,7 @@
 		ValuesFrom "dead"
 	</Result>
 
-	MinPGVersion 80300
+	MinVersion 80300
 </Query>
 
 <Query disk_io>
